### PR TITLE
Feat/lido integration

### PR DESCRIPTION
**Pull request by vgorkavenko**

### CODE REVIEW

File: default.env [hunk 1]
```diff
@@ -156,6 +163,8 @@ EL_NODE=http://execution:8551
 CL_NODE=http://consensus:5052
 # MEV-boost address. This would only be changed for Vouch setups
 MEV_NODE=http://mev-boost:18550
+# Beacon enpoint for OBOL node
+OBOL_NODE_BEACON_ENDPOINT=http://consensus:5052
```
Comment (dgusakov) [inline default.env:167]: If it is the same as CL node why do you introduce a new variable?

Comment (vgorkavenko) [reply default.env:167]: The value for `CL_NODE` will be changed by obol node url if the user selects obol

File: ethd [hunk 2]
```diff
@@ -2409,6 +2497,106 @@ https://0x8c7d33605ecef85403f8b7289c8058f440cbb6bf72b055dfe2f3e2c6695b6a1ea5a9cd
...
 
+query_lido_keys_generation() {
+    __num_validators="1"
+    __keystore_password=""
+    __num_validators=$(whiptail --title "Validators count" --inputbox "Enter the number of validators" 8 60 "${__num_validators}" 3>&1 1>&2 2>&3)
+    while true; do
+        __keystore_password=$(whiptail --title "Keystore password" --passwordbox "Enter validators keystore password (at least 8 chars)" 8 60 "${__keystore_password}" 3>&1 1>&2 2>&3)
+        exitstatus=$?
+        if [ $exitstatus -eq 0 ]; then
+            if [[ ${#__keystore_password} -eq 8 ]]; then
```
Comment (madlabman) [inline ethd]:
```suggestion
            if [[ ${#__keystore_password} -ge 8 ]]; then
```

?

File: ethd [hunk 3]
```diff
@@ -2409,6 +2497,106 @@ https://0x8c7d33605ecef85403f8b7289c8058f440cbb6bf72b055dfe2f3e2c6695b6a1ea5a9cd
...
+        else
+            echo "You chose Cancel."
+            exit 1
+        fi
+    done
+
+
+    exitstatus=$?
+    if [ $exitstatus -eq 0 ]; then
+        echo "your Number of validators is:" "${__num_validators}"
```
Comment (madlabman) [inline ethd]:
```suggestion
        echo "Your number of validators is:" "${__num_validators}"
```

File: ethd [hunk 4]
```diff
@@ -2799,7 +3049,7 @@ if [ "$__result" -eq 127 ]; then
     printhelp
 fi
 
-check_disk_space
+# check_disk_space
```
Comment (madlabman) [inline ethd]: Intended?

File: ethd (hunk 2 above)
Comment (vgorkavenko) [reply ethd]: should be fixed

**Review by madlabman — APPROVED** [review]

**Pull request merged**